pages/storage_and_backup/file_storage/enterprise_file_storage/netapp_revert_snapshot/guide.en-gb.md

```diff
@@ -27,106 +24,120 @@ Snapshots are created from an existing, operational volume. They cannot exist wi
+
+- `{serviceName}` is the service unique ID
 - `{shareId}` is the share to restore
-- `{snapshotID}` is the latest share snapshot 
+- `{snapshotID}` is the latest share snapshot
 
-The OVHcloud API will return a HTTP 202 (Accepted) and no response body.
+The OVHcloud API will return a, HTTP 202 (Accepted) code and no response body.<br>
 The volume status will be set to `reverting` then will go back to `available` once the volume restoration process is complete. At the same time, the snapshot status will be set to `restoring` then will go back to `available` once the volume restoration process is complete.
 
 ![RevertManualSnapshot](images/use_case_1_step_2.png){.thumbnail}
 
+### Scenario 2: Reverting a volume using a snapshot taken via the snapshot policy
 
-## Scenario 2: Revert volume using snapshot taken via snapshot policy
 In this case, a snapshot policy takes regular (automatic) snapshots of a volume. You want to restore your volume to the latest snapshot taken by the policy.
 
-You hold the latest snapshot taken by the snapshot policy associated to a volume. This snapshot becomes `manual`. Once the snapshot has the type `manual`, its volume can be restored to it.
-
-### Requirements
+You will have to hold the latest snapshot taken by the snapshot policy associated to a volume. This snapshot becomes `manual`. Once the snapshot has the type `manual`, its associated volume can be restored to it.
 
-You created a snapshot policy and has associated it to the volume to restore. This snapshot policy has created at least one snapshot.
+> [!primary]
+> **Requirements for this scenario:**
+>
+> - You created a snapshot policy and you associated it to the volume to restore. 
+> - This snapshot policy has created at least one snapshot.
 
 > [!primary]
 >
-> Snapshots taken by the snapshot policy have `automatic` type. In order to be used for volume restoration, they have to be hold using `/hold` API route. This will prevent them from being rotated by the snapshot policy schedule.
+> Snapshots taken by the snapshot policy have an `automatic` type. In order for them to be used for volume restoration, they have to be held using the `/hold` API route. This will prevent them from being rotated by the snapshot policy schedule.
 >
 
-1\. Identify the latest `automatic` volume snapshot using the following API route:
->[!api]
+1\. Identify the latest `automatic` volume snapshot using the following API call:
+
+> [!api]
 >
 > @api {GET} /storage/netapp/{serviceName}/share/{shareId}/snapshot
 >
+
 - `{serviceName}` is the service unique ID
 - `{shareId}` is the share to restore
 
 ![RevertSnapshot](images/use_case_2_step_1.png){.thumbnail}
 
-2\.	Hold the Snapshot using the following API route: 
->[!api]
+2\. Hold the Snapshot using the following API call: 
+
+> [!api]
 >
 > @api {POST} /storage/netapp/{serviceName}/share/{shareId}/snapshot/{snapshotId}/hold
+
 - `{serviceName}` is the service unique ID
 - `{shareId}` is the share to restore
```

Comment:
- `{snapshotId}` is the latest `automatic` snapshot

`{snapshotId}` path parameter description is lacking here after `{shareId}`

Reply:
Added, thanks